portal-web/test/functional/com/liferay/portalweb/macros/analyticscloud/ACNavigation.macro

```diff
@@ -32,6 +32,18 @@ definition {
 		ACNavigation.goToSidebarOption(sidebarOption = "assets");
 	}
 
+	macro goToDefinitionsAttributesTabs {
```

Comment:
Hi @91nilo , I think these 2 macros should be in ACSettings.macro file since most interactions are in AC Settings.